commands/trends_command.py

```diff
@@ -8,12 +8,12 @@
 from boterror import BotError
 from botquery import BotQuery
 from config import BASE_URL
-from constants import SEND_PHOTO_ENDPOINT, PHOTO_KEY, CHAT_ID_KEY, NEW_LINE_CHAR, SPACE, UNDERSCORE, PARSE_MODE_KEY, UNDERSCORE, SPACE
+from constants import SEND_PHOTO_ENDPOINT, PHOTO_KEY, CHAT_ID_KEY, NEW_LINE_CHAR, SPACE, UNDERSCORE, PARSE_MODE_KEY, UNDERSCORE, SPACE, EN_US, TIMEZONE_OFFSET_MINUTES
```

Comment:
Can we rename `EN_US` to LOCALE please 🙏 "en_us" is a locale.

Reply:
will do